ncolony/tests/test_service.py

```diff
@@ -234,10 +234,17 @@ def test_makeservice(self):
         subservices = list(s)
         subservices.remove(pm)
         functions = [subs.call[0] for subs in subservices]
-        paths = set()
+        os.mkdir(self.opt['messages'])
```

Comment:
Why not put this in the `setUp` and then also self `self.mktemp` there to set these paths?